src/FirebaseObject.js

```diff
@@ -335,8 +335,16 @@
             }
 
             var send = $firebaseUtils.debounce(function() {
-              rec.$$scopeUpdated(getScope())
-                ['finally'](function() { sending = false; });
+              var scopeData = getScope();
+              rec.$$scopeUpdated(scopeData)
+                  ['finally'](function() {
+                    sending = false;
+                    if(!scopeData.hasOwnProperty('$value')){
+                      delete rec.$value;
+                      delete parsed(scope).$value;
```

Comment:
These deletes have to happen somewhere. I'm not sure if this is the best location. My other thought was inside the `$$scopeUpdated` function, but that does not have access to the scope value.